utils/jsonapi.js

```diff
@@ -37,6 +36,9 @@ const serializerOptions = (serializerArgs) => {
   const options = {
     pluralizeType: false,
     attributes: resourceKeys,
+    owner: {
+      ref: 'ownerId',
+    },
```

Comment:
Since this is specific to the parks resource, could you find a way to generalize this?